src/docs/src/whatsnew/3.4.rst

```diff
@@ -0,0 +1,236 @@
+* :ghissue:`4653`: Fix ability to use ``;`` inside of config values.
+* :ghissue:`4629`: Fix prometheus to survive ``mem3_sync`` termination.
+* :ghissue:`4626`: Fix purge infos replicating to the wrong shards
+  during shard splitting.
+* :ghissue:`4602`: Fix error handling for the _index endpoint and
+  document ``_index/_bulk_delete``.
+* :ghissue:`4594`: Mango: covering indexes crash on partitioned
+  databases.
+* :ghissue:`4555`: Fix race condition when creating indexes.
+* :ghissue:`4524`: Querying ``_all_docs`` with non-string key should
+  return an empty list.
+* :ghissue:`4514`: GET invalid path under ``_index`` should not cause
+  500 response.
+* :ghissue:`4509`: Make ``remsh`` work with quoted cookie.
+* :ghissue:`4491`: Fix ``couch_index`` to avoid crashes under certain
+  conditions.
+* :ghissue:`4485`: Catch and log any error from ``mem3:local_shards``.
+* :ghissue:`4473`: Fix prometheus counter metric naming.
+* :ghissue:`4458`: Fix text index selection for queries with ``$regex``.
+* :ghissue:`4416`: Allow `_local` doc writes to the replicator dbs.
+* :ghissue:`4370`: Ensure design docs are uploaded individually when
+  replicating with ``bulk_get``.
+* :ghissue:`4363`: Fix replication ``_scheduler/docs`` ``total_rows``.
+* :ghissue:`4360`: Fix handling forbidden exceptions from workers in
+  ``fabric_doc_update``.
+* :ghissue:`4353`: Fix replication ``job_link``.
+* :ghissue:`4348`: Fix undefined function warning in weatherreport.
+* :ghissue:`4343`: Fix undef when parsing replication doc body.
+
+Tests
+-----
+
+* :ghissue:`4873`: Fix test for `text` index creation.
+* :ghissue:`4863`: Fix flaky ``users_db_security`` test.
+* :ghissue:`4808`: Fix flaky ``couch_stream`` test.
+* :ghissue:`4806`: Mango: do not skip json tests when Clouseau installed.
+* :ghissue:`4803`: Fix flaky ``ddoc_cache`` test some more.
+* :ghissue:`4766`: Fix flaky mem3 reshard test.
+* :ghissue:`4763`: Plug hole in unit test coverage of view curser functions.
+* :ghissue:`4726`: Support Elixir 1.15.
+* :ghissue:`4691`: ``make elixir`` should match what we run in CI.
+* :ghissue:`4632`: Fix test database recreation logic.
+* :ghissue:`4630`: Add extra assert in flaky couch_file test.
+* :ghissue:`4620`: Add Erlang/OTP 26 to Pull Request CI matrix.
+* :ghissue:`4552`, :ghissue:`4553`: Fix flaky couchjs error test.
+* :ghissue:`4453`: Fix flaky LRU test taht the new super fast macOS CI worker
+  noticed.
+* :ghissue:`4422`: Clean up JSON index selection and add unit tests.
+* :ghissue:`4346`: Add test coverage for replicator ``user_ctx`` parser.
+
+Docs
+----
+
+* :ghissue:`4774`: Amend description of ``use_index`` on ``/{db}/_find``.
+* :ghissue:`4743`: Ban the laster monster.
+* :ghissue:`4630`: Add ``_design_docs/queries`` and
+  ``_local_docs/queries``.
+* :ghissue:`4645`: Add authenication data to examples.
+* :ghissue:`4636`: Clarify default quorum calculation.
+* :ghissue:`4551`: Clarify encoding length in performance.rst.
```

Comment:
This one leads to a closed pr: https://github.com/apache/couchdb/pull/4551 I think we wanted? https://github.com/apache/couchdb/pull/4561

We could probably skip the `.rst` suffix, that's internal detail. Just the `performance section` or something like that?